go/client/cmd_encrypt.go

```diff
@@ -55,6 +56,10 @@ func NewCmdEncrypt(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comma
 				Usage: "Specify an outfile (stdout by default).",
 			},
 			cli.BoolFlag{
+				Name:  "a, anonymous",
```

Comment:
I'd also prefer to avoid picking a short flag for now, if that's ok with you guys. I prefer not to use short flags that have a common meaning, unless our meaning matches the convention. (`-a` would be "all" or "armor". `-r` would be "recursive" or (standard) "recipient". `-h` would be "help" or maybe "human-readable".)